contracts/TopShotMarketV3.cdc

```diff
@@ -53,7 +54,7 @@ pub contract TopShotMarketV3 {
     /// emitted when the price of a listed moment has changed
```

Comment:
we need this for a specific topshot p2p implementation for now. since we are going to move topshot to storefront eventually I think we'd prefer keep changes minimal to this.